packages/st-lukesmith/st-lukesmith.pacscript

```diff
@@ -1,12 +1,15 @@
 name="st-lukesmith"
 pkgname="st"
-version="0.16.1"
 url="https://github.com/LukeSmithxyz/st.git"
 license="MIT"
 build_depends="make libharfbuzz-bin libxft-dev libpango1.0-dev"
 depends="fontconfig"
 gives="st"
 
+pkgver () {
+  git ls-remote https://github.com/LukeSmithxyz/st.git master | sed "s/refs\/heads\/.*//" # Returns the latest commit hash from remote
```

Comment:
This leaves a tab character at the end, probably better to use cut
`git ls-remote https://github.com/LukeSmithxyz/st.git master | cut -f1`

Reply:
Thanks, I'll update my pacscripts too with that line